### ISSUE TITLE
Absent keratohyalin granules

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6463]
**Preferred term label:**
Absent keratohyalin granules

**Synonyms**

**Definition (free text, please give PubMed ID)**
Lack of keratohyalin granules, which are normally present in the stratum granulosum of the epidermal layer of the skin.

Comment: Keratohyalin granules typically contain a mixture of keratin filament bundles, as well as the filament-associated proteins loricrin, filaggrin, and trichohyalin. These granules are insoluble in water and located within the cytoplasm where they promote dehydration of the cell.  Their chief function seems to be cross-linking of keratin filaments which creates the tight barrier that is the epidermis, providing the body with an impermeable layer that protects from invasion by foreign particles.  This process of cornification of the epidermis effected by KHG is known as keratinization. [PMID:30725734]

**Parent term (use hpo.jax.org/app)**

https://hpo.jax.org/app/browse/term/HP:0011124
-- but instead consider addding subterms for the layers of the epidermis. See also https://onlinelibrary.wiley.com/doi/abs/10.1034/j.1600-0625.2002.110406.x

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

Ichthyosis vulgaris | OMIM:146700

https://pubmed.ncbi.nlm.nih.gov/16444271/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
